apps/hash-frontend/src/pages/entities.page.tsx

```diff
@@ -0,0 +1,220 @@
+import { VersionedUrl } from "@blockprotocol/type-system";
+import { types } from "@local/hash-isomorphic-utils/ontology-types";
+import { isBaseUrl, OwnedById } from "@local/hash-subgraph";
+import { extractBaseUrl } from "@local/hash-subgraph/type-system-patch";
+import {
+  Box,
+  buttonClasses,
+  Container,
+  Typography,
+  useTheme,
+} from "@mui/material";
+import { useRouter } from "next/router";
+import { NextSeo } from "next-seo";
+import { useCallback, useContext, useMemo } from "react";
+
+import { useAccountPages } from "../components/hooks/use-account-pages";
+import { useCreatePage } from "../components/hooks/use-create-page";
+import { EntityTypeEntitiesContext } from "../shared/entity-type-entities-context";
+import { useEntityTypeEntitiesContextValue } from "../shared/entity-type-entities-context/use-entity-type-entities-context-value";
+import { useLatestEntityTypesOptional } from "../shared/entity-types-context/hooks";
+import { AsteriskLightIcon } from "../shared/icons/asterisk-light-icon";
+import { AsteriskRegularIcon } from "../shared/icons/asterisk-regular-icon";
+import { FileCirclePlusRegularIcon } from "../shared/icons/file-circle-plus-regular-icon";
+import { FilesLightIcon } from "../shared/icons/files-light-icon";
+import { FilesRegularIcon } from "../shared/icons/files-regular-icon";
+import { getLayoutWithSidebar, NextPageWithLayout } from "../shared/layout";
+import { HEADER_HEIGHT } from "../shared/layout/layout-with-header/page-header";
+import { tableHeaderHeight } from "../shared/table-header";
+import { Button } from "../shared/ui";
+import { TabLink } from "../shared/ui/tab-link";
+import { Tabs } from "../shared/ui/tabs";
+import { EntitiesTable } from "./shared/entities-table";
+import {
+  TOP_CONTEXT_BAR_HEIGHT,
+  TopContextBar,
+} from "./shared/top-context-bar";
+import { WorkspaceContext } from "./shared/workspace-context";
+
+const contentMaxWidth = 1000;
+
+type ParsedQueryParams = {
+  entityTypeIdOrBaseUrl?: string;
+};
+
+const EntitiesPage: NextPageWithLayout = () => {
+  const router = useRouter();
+  const { activeWorkspace } = useContext(WorkspaceContext);
+
+  const { entityTypeId, entityTypeBaseUrl } = useMemo(() => {
```

Comment:
Why not have `entityTypeId` and `entityTypeBaseUrl` as separate query params? 

I think we only really need `entityTypeBaseUrl` at first (we could be looking for `Page`s by their base URL) but no harm in having `entityTypeId`

Reply:
My thinking was since it shouldn't be possible to allow for both `entityTypeId` and `entityTypeBaseUrl` to be provided, that unifying them in a single query param would make more sense. But I'm happy to change this if you'd prefer it.

Reply:
I don't think it's a big deal. Let's see how the filters evolve